rocketchat/templates/microservices-stream-hub-deployment.yaml

```diff
@@ -0,0 +1,79 @@
+{{- if .Values.microservices.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-stream-hub
+  labels:
+    app.kubernetes.io/name: {{ include "rocketchat.name" . }}
+    helm.sh/chart: {{ include "rocketchat.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    {{- if .Values.deploymentLabels }}
+      {{- toYaml .Values.deploymentLabels | nindent 4 }}
+    {{- end }}
+spec:
+  replicas: {{ .Values.microservices.streamHub.replicas }}
```

Comment:
this is the one that can only ever have one right?  Maybe should hard code value?

Reply:
true! good catch.. gonna to change to hard coded value